encrypt.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+from Crypto import Random
+from Crypto.PublicKey import RSA
+from Crypto.Hash import SHA
+from Crypto.Cipher import AES, PKCS1_OAEP
+import random, string
+import base64
```

Comment:
what are these imports for.No comments found?

Reply:
just cool

Reply:
ok